### ISSUE TITLE
Does it run in Django?

### ISSUE THREAD

**Issue by slkasisto** [dbader/schedule#234]
I have posted a question on StackOverflow:

> https://stackoverflow.com/questions/51427313/scheduling-in-django

which describes what I'm trying to achieve.  Can this library be used in Django, are there any examples of its usage?

Thanks.

**Comment by girisagar46** [comment]
I recommend not using this library in your Django project. 
Possible solution for scheduling task are: 
- Celery, Celery beat 
- [https://github.com/Tivix/django-cron/](https://github.com/Tivix/django-cron/)

Also, ther's answer in your SO question.

**Comment by slkasisto** [comment]
Yes, I figure that by now.  Still would be interesting to see it working or at least understand what is causing it to fail.  I like its API.
